qbft/types.go

```diff
@@ -27,6 +27,10 @@ type Syncer interface {
 	SyncDecidedByRange(identifier types.MessageID, from, to Height)
 }
 
+type RootGetter interface {
```

Comment:
no need for it, you have it in types

Reply:
could not find it, the `RootGetter` interface moved to types package 
in this PR https://github.com/bloxapp/ssv-spec/pull/191

Reply:
under types/messages.go you have an interface called Root. 
No need to duplicate it